attack-theme/static/scripts/matrix.js

```diff
@@ -1,4 +1,8 @@
-load_saved_layout();
+var path = window.location.pathname.toLowerCase();
+
+if (path.indexOf("pre") < 0 && path.indexOf("mobile") < 0) {
+    load_saved_layout();   
+}
```

Comment:
While this works in this instance, I don't think it's an appropriately flexible solution. 
- It's possible a platform could be added to Enterprise which included no sub-techniques. 
- a user could define their own matrix in the config, to be shown under `matrices/super-excellent-matrix/`that included no sub-techniques.
- Mobile could add sub-techniques in the future

So we should avoid making assumptions about which matrices have and don't have sub-techniques based on our knowledge of the _current_ dataset. Instead this should check if the matrices on the page have sub-techniques. The [matrix macro](https://github.com/mitre-attack/attack-website/blob/master/attack-theme/templates/macros/matrix.html#L9) already has an argument for this, so I imagine adding an additional parse-able tag to the page to mark this would be pretty straightforward (just like your `data-selected_layout` property).